tests/kafkatest/tests/streams/base_streams_test.py

```diff
@@ -94,6 +94,6 @@ def wait_for_verification(self, processor, message, file, num_lines=1):
 
     @staticmethod
     def verify_from_file(processor, message, file):
-        result = processor.node.account.ssh_output("grep '%s' %s | wc -l" % (message, file), allow_fail=False)
+        result = processor.node.account.ssh_output("grep -E '%s' %s | wc -l" % (message, file), allow_fail=False)
```

Comment:
I've never seen this before... can you explain what's going on here?

My manpage for grep says that this enables "extended regex", but that in GNU grep, there's no difference between basic and extended. It also looks like "basic" regex means you have to escape '?, +, {, |, (, and )' for them to work as regex operators.

Reply:
oh, I see... it looks like this was left over from a previous iteration of this PR in which you needed regex... Is this change still needed?